packages/react-ui/components/TokenInput/__stories__/TokenInput.stories.tsx

```diff
@@ -183,6 +183,30 @@ export const Validations = () => {
 };
 Validations.storyName = 'validations';
 Validations.parameters = { creevey: { skip: [true] } };
+export const DropdownMenu: Story = () => {
+  return (
+    <Gapped vertical gap={10}>
+      <Wrapper menuWidthMode='stretch' getItems={getItems} />
+    </Gapped>
```

Comment:
Здесь `Gapped` не нужен, т.к. элемент один.
```suggestion
    <Wrapper menuWidthMode='stretch' getItems={getItems} />
```

---

Ещё можно добавить пустую строку перед объявлением этой стори, а то как-то плотничком)